assets/src/dashboard/app/views/editorSettings/components.js

```diff
@@ -109,8 +109,8 @@ export const Error = styled(CenterMutedText)`
 
 export const UploadedContainer = styled.div`
   display: grid;
-  grid-template-columns: repeat(auto-fill, 56px);
-  grid-auto-rows: 56px;
+  grid-template-columns: repeat(auto-fill, 100px);
+  grid-auto-rows: 100px;
```

Comment:
```suggestion
  grid-template-columns: repeat(auto-fill, 102px);
  grid-auto-rows: 102px;
```

To accommodate for the other border as well so images are actually displayed at 96x96